crates/ghost_actor/src/ghost_actor.rs

```diff
@@ -436,14 +465,13 @@ mod tests {
 
             // In this test actor we simply take all the messages we get and
             // add them to our internal state.
-            for mut msg in self.endpoint_as_child.as_mut().drain_messages() {
-                let payload = match msg.take_message().expect("exists") {
+            for (payload, responder) in self.endpoint_as_child.as_mut().drain_messages() {
```

Comment:
I think in practice we should probably still call what's drained a `message`, not a `payload`.